### PR TITLE
Rename/remove rule for package abrt-addon-python

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Remove rule package_abrt-addon-python_removed from RHEL-8 and RHEL-9.
- Add rule package_python3-abrt-addon_removed to RHEL-8.
- 
#### Rationale:

- The package was renamed in RHEL-8 and RHEL-9 has no such package.

### CODE REVIEW

File: linux_os/guide/system/software/system-tools/package_python3-abrt-addon_removed/rule.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {{{ describe_package_remove(package="python3-abrt-addon") }}}
+
+rationale: |-
+    <tt>python3-abrt-addon</tt> contains python hook and python analyzer
+    plugin for handling uncaught exceptions in python programs.
+
+severity: low
+
+identifiers:
+    cce@rhel8: CCE-82923-4
```
Comment (yuumasato) [inline linux_os/guide/system/software/system-tools/package_python3-abrt-addon_removed/rule.yml:17]: I have reused the CCE from RHEL-8, should I actually get a new one?

Comment (jan-cerny) [reply linux_os/guide/system/software/system-tools/package_python3-abrt-addon_removed/rule.yml:17]: I think that yes, you should, it's a different package, therefore a different configuration setting.

Comment (yuumasato) [reply linux_os/guide/system/software/system-tools/package_python3-abrt-addon_removed/rule.yml]: New CCE added.

**Comment by jan-cerny** [comment]
@yuumasato the profile stability test fails

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**